packages/codegen/src/generateEntityCodegenFile.ts

```diff
@@ -85,14 +88,13 @@ type Relation = { kind: "abstract"; line: Code } | { kind: "concrete"; fieldName
 
 /** Creates the base class with the boilerplate annotations. */
 export function generateEntityCodegenFile(config: Config, dbMeta: DbMetadata, meta: EntityDbMetadata): Code {
+  const { entitiesByName: metasByName } = dbMeta;
   const { entity, tagName } = meta;
   const entityName = entity.name;
 
   // Avoid using `do` as a variable name b/c it's a reserved keyword
   const varName = keywords.includes(tagName) ? uncapitalize(entityName) : tagName;
 
-  const metasByName = groupBy(dbMeta.entities, (e) => e.name);
```

Comment:
Unrelated, but `DbMetadata` has this `groupBy` built-in now, so moved to using that.